src/router/SingleModelRouter.js

```diff
@@ -20,10 +20,27 @@ import Router from './Router';
 import { Guard } from '../system';
 
 export default class SingleModelRouter {
+    static createWithModel(model) {
+        let router = new SingleModelRouter(model);
+        return router;
+    }
+
+    static create() {
+        let router = new SingleModelRouter();
+        return router;
+    }
+
+    static createWithRouter(underlyingRouter, modelId) {
```

Comment:
Maybe we don't  need this one as the only one calling it would probably be the `Router.createModelRouter()` function.